index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porofolio main page</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
+</head>
+<body>
+      <header class="head">
+        <nav>
+          <h2 class="pagename" >Welcome</h2>
+        </nav>
+        <img src="tools/Union.svg" alt="" class="nav-icon">
+    </header>
```

Comment:
- [ ] Second Your h2 should not be covered by the `nav` tag and instead of `h2` kindly use `a` tag because this **Welcome** keyword is a link to your homepage 😊
- [ ] First your img should be wrapped with the `nav` tag because `nav` tags contains the links of your mobile menu. Kindly fix it 👍

You can see below the whole solution ⬇️
```
<header class="head">
  <a href="#" class="pagename" >Welcome</a>
  <nav>
    <img src="tools/Union.svg" alt="" class="nav-icon">
  </nav>
</header>
```